packages/pds/src/config/config.ts

```diff
@@ -55,6 +55,15 @@ export const envToCfg = (env: ServerEnvironment): ServerConfig => {
   }
   if (env.blobstoreS3Bucket) {
     blobstoreCfg = { provider: 's3', bucket: env.blobstoreS3Bucket }
+    if (env.blobstoreS3Region) {
+      blobstoreCfg.region = env.blobstoreS3Region
+    }
+    if (env.blobstoreS3Endpoint) {
+      blobstoreCfg.endpoint = env.blobstoreS3Endpoint
+    }
+    if (env.blobstoreS3ForcePathStyle !== void 0) {
```

Comment:
Updating just for consistency with the rest of the codebase.
```suggestion
    if (env.blobstoreS3ForcePathStyle !== undefined) {
```